client/index.html

```diff
@@ -5,44 +5,120 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
         crossorigin="anonymous"></script>
-
-    <title>Woogle</title>
-    <link rel="shortcut icon" href="./woogle.png" type="image/x-icon">
+    <link rel="shortcut icon" href="../client/images/woogle.png" type="image/x-icon">
```

Comment:
Woogle.png doesn't render on index.html unfortunately

Reply:
Ah yeah, need to update that, I did some tweaking on the logo and adjusted the file structure a few times lol